modules/recommended-etcd-practices.adoc

```diff
@@ -64,8 +57,12 @@ $ sudo docker run --volume /var/lib/etcd:/var/lib/etcd:Z quay.io/openshift-scale
 ----
 --
 
-The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile of the fsync metric captured from the run to see if it is less than 10 ms.
+The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile of the fsync metric captured from the run to see if it is less than 20 ms. These are some of the most important etcd metrics that could be potentially affected by I/O performance:
+
+- * `etcd_disk_wal_fsync_duration_seconds_bucket` metric reports the etcd's WAL fsync duration.
```

Comment:
the extra "*" show up in the rendered docs here.  looks like the leading "-" should be enough.

Reply:
fixed